addons/danger/functions/fnc_immediateActionExp.sqf

```diff
@@ -13,23 +13,27 @@
 // init
 params ["_unit","_pos"];
 
-// Standing or recent explosions ignored
-if !(stance _unit isEqualTo "STAND" || {_unit getVariable [QGVAR(isExploded),0] > time}) exitWith {false};
+// Standing or recent explosions ignored // probably want to add more checks for captive states or similar static behaviour.
+if (
+    isplayer _unit 
+    || {stance _unit isEqualTo "PRONE"}
+    || {_unit getVariable [QGVAR(isExploded),0] > time}
+) exitWith {false};
```

Comment:
This file is WIP. 
Discussion at https://github.com/nk3nny/LambsDanger/issues/37

Reply:
As discussed, this should be most likely be removed or off loaded into its own addon module.